packages/story-editor/src/karma/fixture/fixture.js

```diff
@@ -1040,6 +1040,40 @@ class APIProviderFixture {
           );
         }
 
+        // If we're getting custom fonts, return custom only.
+        if (params.service === 'custom') {
```

Comment:
Why not just

```js
if (params.service) {
	fonts = fonts.filter(({ service }) => service === params.service);
}
```

Reply:
For a moment I thought that the custom fonts should not get included into `fonts` generally, but they should. That's why :) Will update and add some custom fonts to the general fonts list and use filtering.

Reply:
Changed. Looks like `builtin` is used as the service value in API, so can't match it directly to `font.service` here.